packages/catalog-model/src/schema/kinds/Template.v1beta2.schema.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "TemplateV1beta1",
+  "description": "A Template describes a scaffolding task for use with the Scaffolder. It describes the required parameters as well as a series of steps that will be taken to execute the scaffolding task.",
+  "examples": [
+    {
+      "apiVersion": "backstage.io/v1beta1",
+      "kind": "Template",
+      "metadata": {
+        "name": "react-ssr-template",
+        "title": "React SSR Template",
+        "description": "Next.js application skeleton for creating isomorphic web applications.",
+        "tags": ["recommended", "react"]
+      },
+      "spec": {
+        "owner": "artist-relations-team",
+        "type": "website",
+        "parameters": {
+          "required": ["name", "description"],
+          "properties": {
+            "name": {
+              "title": "Name",
+              "type": "string",
+              "description": "Unique name of the component"
+            },
+            "description": {
+              "title": "Description",
+              "type": "string",
+              "description": "Description of the component"
+            }
+          }
+        }
+      }
+    }
+  ],
+  "allOf": [
+    {
+      "$ref": "Entity"
+    },
+    {
+      "type": "object",
+      "required": ["spec"],
+      "properties": {
+        "apiVersion": {
+          "enum": ["backstage.io/v1beta2"]
+        },
+        "kind": {
+          "enum": ["Template"]
+        },
+        "metadata": {
+          "type": "object",
+          "properties": {
+            "title": {
+              "type": "string",
+              "description": "The nice display name for the template. This field is required as is used to reference the template to the user instead of the metadata.name field.",
+              "examples": ["React SSR Template"],
+              "minLength": 1
+            }
+          }
+        },
+        "spec": {
+          "type": "object",
+          "required": ["type", "steps"],
+          "properties": {
+            "type": {
```

Comment:
Not sure we should keep this one. We need to look at what dimensions that people want to filter by and only keep those, perhaps implementing them differently. Only using tags, for example? Or have a `category` string field that you can use any way you like?

Either way, the wording of the description makes it sound like it's copy pasted from (or tailored for) only components and I believe that this will not be our only use case so it would be risky to dig ourselves into that particular hole at this point.

I vote to leave out this field entirely if we can, and introduce later if we need to.

Reply:
Has some coupling with the frontend atm, but I think we should get rid of it idd

Reply:
kk, let's keep track of that somehow